src/service/position-observer-impl.js

```diff
@@ -235,9 +235,7 @@ export class AmpDocPositionObserver extends AbstractPositionObserver {
       }
       timeout = setTimeout(stopScroll.bind(this), 500);
     }));
-    this.unlisteners_.push(this.viewport_.onChanged(() => {
-      // TODO (@zhouyx, #9208): Consider doing this only when event.relayoutAll
-      // is true.
+    this.unlisteners_.push(this.viewport_.onResize(() => {
```

Comment:
Removed the TODO, we actually wanna always do this, not just when relayoutAll is true @zhouyx